nameable.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
```

Comment:
- [ ] It would be nice if you remove the commented piece of code to follow the [best practices of Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md) to follow the requirements of this milestone 👍